src/Controller/TrickController.php

```diff
@@ -30,9 +31,12 @@ class TrickController extends AbstractController
      * @param EntityManagerInterface $em
      * @return Response
      */
-    public function add(Request $request, SluggerInterface $slugger, EntityManagerInterface $em){
+    public function add(Request $request, SluggerInterface $slugger, EntityManagerInterface $em): Response
+    {
         $trick = new Trick;
 
+        $this->denyAccessUnlessGranted('MANAGE', $trick);
```

Comment:
`$this->denyAccessUnlessGranted('ROLE_USER');`

Reply:
modifié